### ISSUE TITLE
[BUG] HTML Interpretation on Paste

### ISSUE THREAD

**Issue by unKn0wnUser453** [giuspen/cherrytree#1317]
**Version, Operative system**
CherryTree 0.99.18, Debian Linux

**Describe the bug**
HTML is not being interpreted correctly on paste
This was supported/worked in the past, however it does not seen to work anymore.
When copying HTML text and pasting it into CherryTree, it does not convert to richtext, however it pastes a single character "r". 

**HTML (Clipboard):**
```
<pre><font color="#729FCF"><b>====================================( </b></font><font color="#8AE234"><b>Test</b></font><font color="#729FCF"><b> )====================================</b></font>
</pre>
```
**On Paste:**
```
<rich_text>r</rich_text>
```
**On Paste (Previously):**
```
</rich_text><rich_text foreground="#729fcf" weight="heavy">====================================( </rich_text><rich_text foreground="#8ae234" weight="heavy">Test</rich_text><rich_text foreground="#729fcf" weight="heavy"> )====================================</rich_text><rich_text>
```

When copied from https://www.tutorialspoint.com/online_html_editor.php it seems to work when copying the interpreted HTML from that.

I'm not sure if this issue is relevant to the issue:
https://github.com/giuspen/cherrytree/issues/450

**Comment by ciranus** [comment]
I do not confirm

`    <pre><font color="#729FCF"><b>====================================( </b></font><font color="#8AE234"><b>Test</b></font><font color="#729FCF"><b> )====================================</b></font>
</pre>`

Thunderbird html viewer:
![2020-11-06_12-55](https://user-images.githubusercontent.com/61602511/98363550-6a114300-202f-11eb-8c53-ce342339275b.png)

Copy/paste to 0.99.18:
![2020-11-06_12-56](https://user-images.githubusercontent.com/61602511/98363634-9331d380-202f-11eb-95bd-86c72a88b4d1.png)

exported to xml:
```
    <rich_text foreground="#729fcf" weight="heavy">====================================( </rich_text>
    <rich_text foreground="#8ae234" weight="heavy">Test</rich_text>
    <rich_text foreground="#729fcf" weight="heavy"> )====================================</rich_text>
    <rich_text>
</rich_text>
```

**Comment by ciranus** [comment]
@unKn0wnUser453
I cannot reproduce your issue in Debian Bullseye (= testing)
Can you say more on your issue:

1 - CT 0.99.18 cannot currently be directly installed in Debian Buster (= stable): see [#1315](https://github.com/giuspen/cherrytree/issues/1315)
=> can you show this feedback: 
```shell
dpkg -l cherrytree libc6 libgcc-s1 libglibmm-2.4-1v5 libstdc++6 | awk '/^i/ {print $2,$3}'
```
2 - you copy from which application to reproduce your issue ?

3 - can you try with Cherrytree 0.99.**19** to confirm ?

**Comment by txe** [comment]
@unKn0wnUser453 , can you give the better example? I don't understand from where you took
```
<pre><font color="#729FCF"><b>====================================( </b></font><font color="#8AE234"><b>Test</b></font><font color="#729FCF"><b> )====================================</b></font>
</pre>
```
Also, I could copy text from https://www.tutorialspoint.com/online_html_editor.php though it looked as a plain text when it was pasted.

**Comment by ciranus** [comment]
No problem at all for me from https://www.tutorialspoint.com/online_html_editor.php.

online_html_editor html code:
![2020-11-08_14-11](https://user-images.githubusercontent.com/61602511/98465902-7252b500-21cc-11eb-8ceb-70cd68248d78.png)

Copy/paste from online_html viewer to CT works as expected.
![2020-11-08_14-14](https://user-images.githubusercontent.com/61602511/98465951-bb0a6e00-21cc-11eb-936f-eda20de77ad2.png)

**Comment by unKn0wnUser453** [comment]
So for debugging I tried building cherrytree from the repository and using the dpkg, both come up with same result.
dpkg-query: no packages found matching cherrytree
libc6:amd64 2.31-4
libc6:i386 2.31-4
libgcc-s1:amd64 10.2.0-15
libgcc-s1:i386 10.2.0-15
libglibmm-2.4-1v5:amd64 2.64.2-2
libstdc++6:amd64 10.2.0-15

So what I am attempting to do is copy colored-text from the terminal and copy it into cherrytree keeping the colored text. Only way to do so is with "Copy as HTML" from the terminal (in this case I am using gnome-terminal) and pasting it into cherrytree. It did work before in previous versions of cherrytree.

For some reason when copying into cherrytree it only copies an "r" and nothing else. 

Steps:
sudo apt-get install gnome-terminal
sudo apt-get install cherrytree
gnome-terminal
cherrytree
copy colored text from the terminal as "Copy as HTML"
create a tree and paste into cherrytree (do not paste as plain text)

**Comment by txe** [comment]
@unKn0wnUser453 , thanks, I can reproduce the issue now

**Comment by unKn0wnUser453** [comment]
@txe Glad someone was able to reproduce! I was worried it was just me.

**Comment by txe** [comment]
Fixed. The fix will be in 0.99.20.
The issue: the text in clipboard had UTF-16 BOM encoding instead of UTF-8 and cherrytree couldn't handle it.
